test/ci_support/example_test_failure_github_issue.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+
+#
+# Implementation code
+#
+
+# Find the implementation
+tribitsDir = os.environ.get("TRIBITS_DIR", "")
+if tribitsDir:
+  sys.path = [os.path.join(tribitsDir, "ci_support")] + sys.path
+else:
+  raise Exception("ERROR, TRIBITS_DIR must be set in the env to 'tribits' base dir!")
+
+import CreateIssueTrackerFromCDashQuery as CITFCQ
+
+
+usageHelp = \
+r"""Stock error message.
+"""
+
+
+# The main function
+def main():
+  issueTrackerCreator = \
+    CITFCQ.CreateIssueTrackerFromCDashQueryDriver(
+      ExampleIssueTrackerFormatter(),
+      cdashProjectStartTimeUtc="6:00", # Midnight MST
+      usageHelp=usageHelp,
+      issueTrackerUrlTemplate="https://github.com/<group>/<group>/issues/<newissueid>",
+      issueTrackerTemplate="#<newissueid>" )
+  issueTrackerCreator.runDriver()
+
+
+# Nonmember function to actually create the body of the new GitHub marddown text
+#
+# NOTE: This was made a nonmember function to put this to the bottom and not
+# obscure the implementing class 'ExampleIssueTrackerFormatter'
+#
+def getGithubIssueBodyMarkdown(
```

Comment:
Is the indentation inside this function correct?

Reply:
It used one space instead of two spaces by accident.  It is fixed in my local branch.

NOTE: The raw text lines are not indented to avoid indent in the generated text.  I don't love doing this but I don't know of a better solution in Python that would not be silly.

Reply:
I don't know if it applies here, but I tend to use `textwrap.dedent()` often enough.

Reply:
I did not know about `textwrap.dedent()` but it does not seem to work as I expected.  It does not seem to actually remove the indentation with Python 2.7 or 3.5.2.

I wonder if it gets messed up with lines with all bank spaces?

Reply:
Oh, perhaps.